javalin/src/test/java/io/javalin/TestWebSocket.kt

```diff
@@ -547,7 +555,7 @@ class TestWebSocket {
             if (System.currentTimeMillis() > limitTime) {
                 throw TimeoutException("Wait for condition has timed out")
             }
-            Thread.sleep(2)
+            Thread.sleep(50)
```

Comment:
This seems like it would just unnecessarily slow things down?

Reply:
Not really, in my case this test requires a little bit more time:

![obraz](https://user-images.githubusercontent.com/4235722/174443889-aa2af9fd-f652-4dda-bb5a-fba0ecc7810a.png)

As it randomly throws, so looks like the disconnect/timeout is called before Jetty processes the value.